apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/_components/Sidebar/PublishDraftCommitModal/index.tsx

```diff
@@ -312,23 +325,48 @@ export default function PublishDraftCommitModal({
       title='Publish new version'
       description='Publishing the version will publish all changes in your prompts to production. Review the changes carefully before publishing.'
       onOpenChange={() => onClose(null)}
-      onConfirm={() => publishDraft({ projectId: project.id, id: commitId! })}
+      onConfirm={() =>
+        publishDraft({
+          projectId: project.id,
+          id: commitId!,
+          title,
+          description,
+        })
+      }
       confirm={{
         label: isLoading ? 'Validating...' : 'Publish to production',
         description: confirmDescription({ isLoading, anyChanges, hasErrors }),
         disabled: isLoading || hasErrors,
         isConfirming: isPublishing,
       }}
     >
-      <ChangesList
-        anyChanges={anyChanges}
-        commit={commit}
-        projectId={project.id}
-        isLoading={isLoading}
-        groups={groups}
-        hasErrors={!isLoading && hasErrors}
-        onClose={onClose}
-      />
+      <div className='flex flex-col gap-4'>
+        <FormWrapper>
```

Comment:
FormWrapper already has space of 4. Just add there the `flex flex-col`

Reply:
nah that is there because of the changes list which makes sense to be it separate from fromwrapper

Reply:
But is the same vertical space as if you put inside form wrapper no?